### PR TITLE
Add more details about starting up an islandora.

### PR DESCRIPTION
**Pull request by rosiel**
## Purpose / why

The "demo" section was really sparse, and there are several ways to quickly try out an islandora. i know this'll be updated again come ISLE, but I think it's useful to have documented! (todo also: improve the installation section to match/be updated).

## What changes were made?

Replaced a lonely link to the future.islandora.ca sandbox, and a complex way of installing with vagrant (which didn't do what we thought it did, since the islandora base box!) with a more fulsome and up-to-date version, also mention the VMs!

## Verification

Uh, links work? Instructions work? Statements accurate? Etc?

## Interested Parties

@manez @DonRichards  (pinging for interest in documentation! :) ) @dannylamb 

## Related issue:

- this documents #1494.
---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [ ] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [ ] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [ ] Are the changes accurate, useful, free of typos, etc?

> __Person merging__ should ensure the following
* [ ] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [ ] If pages are renamed or removed, have all internal links to those pages been fixed?
* [ ] If pages are added, have they been linked to or placed in the menu?
* [ ] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -17,40 +17,40 @@ Islandora:
...
-```
-CentOS 7:
-```console
-$ git clone https://github.com/Islandora-Devops/islandora-playbook
-$ cd islandora-playbook
-$ vagrant plugin install vagrant-vbguest
-$ ISLANDORA_DISTRO="centos/7" vagrant up
+### Islandora Base Box via Vagrant
+
+If you have Vagrant and Git installed as well as VirtualBox, you can spin up the latest release of Islandora Defaults using the [Islandora Playbook](https://github.com/Islandora-Devops/islandora-playbook)'s Vagrantfile. Out of the box, `vagrant up` will download the latest release of Islandora as the [Islandora 8 Base Box](https://app.vagrantup.com/islandora/boxes/8). It will also set up shared folders and port forwarding, but (as of June 2020) will not do any Ansible Provisioning, because the Islandora 8 Base Box has been already provisioned. Same credentials as above, with further documented in its README file.
```
Comment (rosiel) [inline docs/index.md]: I just learned, in making #1598 , that you also have to change to the dev branch!

File: docs/index.md [hunk 2]
```diff
@@ -17,40 +17,40 @@ Islandora:
...
 - **Is a community** - A [dedicated, active community of users and developers](https://groups.google.com/forum/#!forum/islandora) is working to push new features, collaborate on improvements, design custom solutions, and create extended functionality. Some of these for Islandora 8 take the form of [Recipes](https://github.com/Islandora-Labs/Islandora-Cookbook).
 
-## Demo
+## Try Islandora
 
-Visit [http://future.islandora.ca/](http://future.islandora.ca/).
+### Sandbox
 
-## Quickstart
+To try Islandora without installing anything, visit [http://future.islandora.ca/](http://future.islandora.ca/). Anyone can log in to this sandbox as an administrator (credentials are below and on the front page) and explore the interface! However, this site is refreshed periodically so your changes will not be permanent. This site uses Islandora Defaults, a way of setting up Islandora for demonstration purposes - this is not the only way that Islandora can be made to work!
```
Comment (manez) [inline docs/index.md]: Add "This sandbox also demonstrates some Drupal contrib modules that are not standard in Islandora, such as View Bulk Edit, which powers the Batch Edit Page."

File: docs/index.md [hunk 3]
```diff
@@ -17,40 +17,40 @@ Islandora:
...
 
+### Virtual Machine Image
 
-### Requirements
-- [Virtual Box](https://www.virtualbox.org/)
-- [Vagrant](https://www.vagrantup.com/) (version 2.0 or higher required)
-- [Git](https://git-scm.com/)
-- [OpenSSL](https://www.openssl.org/)
-- [Ansible](https://www.ansible.com/community) (up to, and not past, 2.8.7)
+To try Islandora locally, you can download and run the latest [community sandbox VM image](https://islandora.ca/try). This requires installing VirtualBox or another virtualization provider. This also uses Islandora Defaults, as it is a pre-built version of the Islandora Playbook.
```
Comment (manez) [inline docs/index.md]: Add "along with sample content, Views, Blocks, and Drupal contributed modules and themes, to enhance its role as a sandbox."

**Review by manez — CHANGES_REQUESTED** [review]
The sandbox and the VM both have some extra bells and whistles beyond Defaults, which we should probably mention so folks don't expect them in the ISLE or Ansible builds where they don't show up.

File: docs/index.md [hunk 4]
```diff
@@ -11,46 +11,48 @@ Islandora:
...
 - **Is a community** - A [dedicated, active community of users and developers](https://groups.google.com/forum/#!forum/islandora) is working to push new features, collaborate on improvements, design custom solutions, and create extended functionality. Some of these for Islandora 8 take the form of [Recipes](https://github.com/Islandora-Labs/Islandora-Cookbook).
 
-## Demo
+## Try Islandora
 
-Visit [http://future.islandora.ca/](http://future.islandora.ca/).
+### Sandbox
 
-## Quickstart
+To try Islandora without installing anything, visit [http://future.islandora.ca/](http://future.islandora.ca/). Anyone can log in to this sandbox as an administrator (credentials are below and on the front page) and explore the interface! However, this site is refreshed periodically so your changes will not be permanent. This site uses Islandora Defaults, a way of setting up Islandora for demonstration purposes. This is not the only way that Islandora can be made to work! This sandbox includes some sample content and also demonstrates some Drupal contrib modules that are not "standard" Islandora, such as View Bulk Edit, which powers the Batch Edit Page.
```
Comment (manez) [inline docs/index.md]: Sorry, typo in my suggestion earlier. The module is called "Views Bulk Edit" not "View Bulk Edit"

**Review by manez — APPROVED** [review]

**Comment by manez** [comment]
Tested the instructions and reviewed the wording. All good!

**Pull request merged**